exporter/coralogixexporter/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+type: coralogix
+
+status:
+  class: exporter
+  stability:
+    beta: [traces, metrics]
+    alpha: [logs]
+  distributions: [contrib, observiq]
```

Comment:
Out of ignorance, why is `observiq` included in the distributions list here, vs. any other company's distribution? I assume there's some historical context here, I'm just not familiar.

Reply:
History i here: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21383.  It is up to the distribution owners if they want to list their distribution.

Reply:
Appreciate the context, thank you!